pkg/cmd/extension/manager.go

```diff
@@ -328,6 +330,15 @@ func (m *Manager) installBin(repo ghrepo.Interface, target string) error {
 	return nil
 }
 
+func generateMissingBinaryIssueCreateCommand(repoOwner string, repoName string, currentPlatform string) string {
```

Comment:
I've moved this into its own method because it is a bit tidier, and I can reuse this method in #9599.